frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -83,34 +52,50 @@ const getIconClass = (status: ClusterUpdateStatus) => {
   }[status];
 };
 
-const FailedConditionAlert = ({message, condition}) => <div className="alert alert-danger">
-  <i className="pficon pficon-error-circle-o" aria-hidden="true" /> <strong>{message}</strong> {condition.message}
+const UpdateFailingAlert: React.SFC<UpdateFailingAlertProps> = ({condition, updatesAvailable=false}) => <div className="alert alert-danger">
+  <i className="pficon pficon-error-circle-o" aria-hidden="true" /> <strong>Update is failing.</strong> {condition.message}.
+  &nbsp;
+  View <Link to="/settings/cluster/clusteroperators?orderBy=desc&sortBy=Status"> Cluster Operators </Link> for details {updatesAvailable ? ' or check available updates to try another version.' : '.'}
```

Comment:
Will this possibly result in `View Cluster Operators for details .` (space period)

Reply:
Fixed